sky/skylet/providers/aws/config.py

```diff
@@ -1001,49 +1001,23 @@ def _update_inbound_rules(target_security_group, sgids, config):
     extended_rules = (
         config["provider"].get("security_group", {}).get("IpPermissions", [])
     )
-    user_specified_rules = _retrieve_user_specified_rules(
-        config["provider"].get("ports", [])
-    )
-    ip_permissions = _create_default_inbound_rules(
-        sgids, user_specified_rules, extended_rules
-    )
+    ip_permissions = _create_default_inbound_rules(sgids, extended_rules)
```

Comment:
Please help update #1702, once this PR is merged.

Reply:
sure! 🫡